.gitignore

```diff
@@ -7,7 +7,8 @@
 node_modules
 #files from platform that should be ignored (show as changed because of www-data ownership)
 /__init__.py
-/jsconfig.json
```

Comment:
should probably leave jsconfig.json in to handle older versions of platform, no?

Reply:
Sure, I'll add it back